src/components/functional_module/include/functional_module/generic_module.h

```diff
@@ -60,7 +60,7 @@ typedef utils::SharedPtr<GenericModule> ModulePtr;
 struct PluginInfo {
   std::string name;
   int version;
-  std::deque<MobileFunctionID> mobile_function_list;
+  std::deque<RCFunctionID> mobile_function_list;
```

Comment:
@LitvinenkoIra 
If you renamed enum you have to rename this member to keep consistency